### PR TITLE
Ignore assets with collectible ids in db upgrade

### PR DESCRIPTION
**Pull request by nicholasyoder**
Closes https://github.com/orgs/rotki/projects/11?pane=issue&itemId=96870115

### CODE REVIEW

File: rotkehlchen/db/upgrades/v46_v47.py [hunk 1]
```diff
@@ -72,7 +72,8 @@ def _remove_nft_collection_assets(write_cursor: 'DBCursor') -> None:
                 'SELECT identifier FROM assets WHERE identifier IN ('
                 "SELECT identifier FROM evm_tokens WHERE token_kind = 'B' "
                 "UNION SELECT REPLACE(identifier, 'erc721', 'erc20') "
-                "FROM evm_tokens WHERE token_kind = 'B');",
+                "FROM evm_tokens WHERE token_kind = 'B') AND "
+                "LENGTH(identifier) - LENGTH(REPLACE(identifier, '/', '')) = 1;",
```
Comment (LefterisJP) [inline rotkehlchen/db/upgrades/v46_v47.py:76]: Isn't the length standard? I mean the length of the token identifiers for NFTs ... unless I recall wrong aren't they always the same? So can't you check if it's bigger than this standard length? (along with a comment explaining what you are doing here.

File: rotkehlchen/tests/db/test_db_upgrades.py [hunk 2]
```diff
@@ -2744,6 +2744,10 @@ def _check_tokens_existence(user_db_cursor: 'DBCursor', expect_removed: bool = F
                 'SELECT COUNT(*) FROM assets WHERE identifier IN (?, ?)',
                 (uniswap_erc20_token.identifier, uniswap_erc721_token.identifier),
             ).fetchone()[0] == 0 if expect_removed else 2
+            assert global_db_cursor.execute(
+                'SELECT COUNT(*) FROM assets WHERE identifier = ?',
+                (erc721_token_with_id.identifier,),
+            ).fetchone()[0] == 1  # tokens with collectible ids shouldn't be modified
```
Comment (LefterisJP) [inline rotkehlchen/tests/db/test_db_upgrades.py:2750]: did you add it yourself in the DB or was it added before?

File: rotkehlchen/db/upgrades/v46_v47.py (hunk 1 above)
Comment (nicholasyoder) [reply rotkehlchen/db/upgrades/v46_v47.py]: yes, that will make it simpler.

File: rotkehlchen/tests/db/test_db_upgrades.py (hunk 2 above)
Comment (nicholasyoder) [reply rotkehlchen/tests/db/test_db_upgrades.py:2750]: Its added here along with the others used in the test: https://github.com/rotki/rotki/blob/6bfb21bfb74c2548a646e301d32cb9fc161b69fe/rotkehlchen/tests/db/test_db_upgrades.py#L2780

We have to add these within the test as none of these are assets that we want to actually have in the packaged globaldb.

**Review by LefterisJP — APPROVED** [review]
LGTM
